packages/airnode-node/test/mock-utils.ts

```diff
@@ -61,3 +61,16 @@ export const createAndMockGasTarget = (txType: 'legacy' | 'eip1559') => {
 
   return { gasTarget: { maxPriorityFeePerGas, maxFeePerGas }, blockSpy, gasPriceSpy };
 };
+
+/**
+ * Mocks the console for silenced tests and otherwise enables it for local development.
+ */
+export const mockConsole = () => {
```

Comment:
This solution seems to allow using `console.log` locally and still passes Github tests. Other solutions (mostly) completely disable the console, and then try to restore it if logging is needed, or implement a custom logger (`process.stdout.write` seems to still work).

I think this `mockConsole` approach may be enough for this task. What do you think @Siegrift ?